src/data/store.js

```diff
@@ -0,0 +1,11 @@
+import { applyMiddleware, createStore } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+
+import rootReducer from './reducers';
+
+const store = createStore(
+  rootReducer,
```

Comment:
We pass our `rootReducer` in here so the store knows where its data is coming from.